src/extensions/service-managers/van-api-keys-manager/index.js

```diff
@@ -0,0 +1,70 @@
+/// All functions are OPTIONAL EXCEPT metadata() and const name=.
+/// DO NOT IMPLEMENT ANYTHING YOU WILL NOT USE -- the existence of a function adds behavior/UI (sometimes costly)
+
+import { getFeatures } from "../../../server/api/lib/config";
+import { cacheableData, r } from "../../../server/models";
+import { getConfig } from "../../../server/api/lib/config";
+import { convertSecret } from "../../secret-manager";
+
+export const name = "van-api-keys-manager";
+
+export const metadata = () => ({
+  // set canSpendMoney=true, if this extension can lead to (additional) money being spent
+  // if it can, which operations below can trigger money being spent?
+  displayName: "VAN setup manager",
+  description: "Used for VAN integration without environment variables",
+  canSpendMoney: false,
+  moneySpendingOperations: ["onCampaignStart"],
```

Comment:
if `canSpendMoney` is false, why is there a function listed in the `monetSpendingOperations` list?